openproblems/tasks/dimensional_reduction/examples/citeseq_rmse/scanpy_processing.py

```diff
@@ -0,0 +1,112 @@
+import scanpy as sc
+
```

Comment:
What is the purpose of this file?

Reply:
to keep the example notebook clean. open to suggestions for other solutions